src/components/authRequired/authRequired.js

```diff
@@ -5,7 +5,13 @@ import { Redirect } from 'react-router-dom';
 import LoadingSpinner from '../LoadingSpinner';
 
 export default () => Component => {
-  function authRequired({ authenticated, error, loading, ...passThrough }) {
+  function authRequired({
```

Comment:
you can probably remove dispatch from the props, since you're not going down this road.